### PR TITLE
Replace NGLview with images or gifs

### LINKED ISSUE #255: [DEPLOYMENT] Size of artifacts deployed too large causing timeouts
We're deploying 8.4 GB of artifacts when trying to deploy, that's causing the deployment action to timeout (rightfully).

We need to find a proper solution to this problem.

### LINKED ISSUE #257: Mention conda environments in installation instructions
To reduce installation issue, it might be good to tell the users to (optionally) create a fresh `conda` environment for MDAnalysis.

https://userguide.mdanalysis.org/stable/installation.html#conda

### LINKED ISSUE #259: Broken link to AutoDock specs for PDBQT
The PDBQT specification contains a broken link to AutoDock "What is the format of a PDBQT file".

https://autodock.scripps.edu/faqs-help/faq/what-is-the-format-of-a-pdbqt-file

### LINKED ISSUE #262: CI tests are failing
See: https://github.com/MDAnalysis/UserGuide/actions/runs/5515357882/jobs/10055602253

### PR DESCRIPTION
**Pull request by lilyminium**
Fixes #255
Fixes #257 
Fixes #259

Unfortunately replacing NGLView widgets with images is difficult to automate; every once in a while we might need to uncomment the nglview widgets to check they match the embedded images.

### CODE REVIEW

**Comment by IAlibay** [comment]
What's the new size gh-pages size after all this?

**Comment by lilyminium** [comment]
The sum of the build directory is 86 MB. The images should be summing to ~20 MB of that, although I also decreased to size of some that already existed, so the delta should be a bit smaller. It would be difficult to reduce further than that without removing images or seriously decreasing their quality -- at that point I would be keen to look at non `gh-pages` solutions.

**Comment by lilyminium** [comment]
Bypassed the issue in #262 by limiting it to top 3 h-bonds for now.

File: doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb [hunk 1]
```diff
@@ -21,6 +21,9 @@
     "    \n",
```
Comment (IAlibay) [inline doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb:21]: <pre class="ql-syntax" spellcheck="false">Line #4.    # import nglview as nv
</pre><p>Any specific reason for disabling the imports completely?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/#comment-1272125587'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

File: doc/source/examples/analysis/alignment_and_rms/rmsf.ipynb [hunk 2]
```diff
@@ -18,8 +18,12 @@
     "* MDAnalysis (<a data-cite=\"michaud-agrawal_mdanalysis_2011\" href=\"https://doi.org/10.1002/jcc.21787\">Michaud-Agrawal *et al.*, 2011</a>, <a data-cite=\"gowers_mdanalysis_2016\" href=\"https://doi.org/10.25080/Majora-629e541a-00e\">Gowers *et al.*, 2016</a>)\n",
```
Comment (IAlibay) [inline doc/source/examples/analysis/alignment_and_rms/rmsf.ipynb:17]: <pre class="ql-syntax" spellcheck="false">Line #4.    import nglview as nv
</pre><p>Following from the above - it's not commented out here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/#comment-1272125591'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

File: doc/source/examples/analysis/hydrogen_bonds/hbonds-lifetimes.ipynb [hunk 3]
```diff
@@ -100,7 +100,7 @@
     {
```
Comment (IAlibay) [inline doc/source/examples/analysis/hydrogen_bonds/hbonds-lifetimes.ipynb:100]: <p>Why the change here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/#comment-1272125594'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

**Review by IAlibay — APPROVED** [review]
thanks! overall lgtm, but I do have a couple of questions. I will approve eagerly for now since I don't think any of them are blockers.

File: doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb (hunk 1 above)
Comment (lilyminium) [reply doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb:21]: Just that if people don't want to install nglview, they don't need to.

File: doc/source/examples/analysis/hydrogen_bonds/hbonds-lifetimes.ipynb (hunk 3 above)
Comment (lilyminium) [reply doc/source/examples/analysis/hydrogen_bonds/hbonds-lifetimes.ipynb:100]: This was fixing a previous issue that's now been superceded by #264

File: doc/source/examples/analysis/alignment_and_rms/rmsf.ipynb (hunk 2 above)
Comment (lilyminium) [reply doc/source/examples/analysis/alignment_and_rms/rmsf.ipynb:17]: my mistake -- fixed
